src/node_contextify.cc

```diff
@@ -140,7 +140,7 @@ BaseObjectPtr<ContextifyContext> ContextifyContext::New(
   if (!(CreateV8Context(env->isolate(), object_template, snapshot_data, queue)
             .ToLocal(&v8_context))) {
     // Allocation failure, maximum call stack size reached, termination, etc.
-    return BaseObjectPtr<ContextifyContext>();
+    return nullptr;
```

Comment:
I think I'd prefer `return {};` for the pattern here using the default constructor. That would match the style we've used elsewhere and, IMO makes things a bit clearer.

Reply:
IMO `BaseObjectPtr` is similar to a smart pointer, i.e. `shared_ptr`, and `nullptr` shall be prefered to represent a null pointer.

Reply:
I'd agree with @jasnell but that also extends to keeping it consistent with `shared_ptr`, `weak_ptr` and friends 🙂

Reply:
Yeah, consider it a non-blocking nit. I rather which the `shared_ptr`, etc cases could use the `return {}` approach also.

Reply:
I'll keep this as is for now, consistent with `shared_ptr` and friends. Let's discuss this rule in a follow-up discussion.